transferui/src/main/java/com/hyperwallet/android/ui/transfer/viewmodel/ListTransferDestinationViewModel.java

```diff
@@ -103,6 +114,13 @@ public void loadTransferDestinationList() {
             @Override
             public void onTransferMethodListLoaded(List<TransferMethod> transferMethods) {
                 if (transferMethods != null) {
+                    if (mIsSourcePrepaidCard) {
+                        for (int i = transferMethods.size() - 1; i >= 0; --i) {
+                            if (Objects.equals(transferMethods.get(i).getField(TYPE), PREPAID_CARD)) {
+                                transferMethods.remove(i);
+                            }
+                        }
+                    }
```

Comment:
This might throw `ConcurrentModificationException`, use iterator instead

Reply:
fixed